spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/src/test/resources/application-test.properties

```diff
@@ -0,0 +1,2 @@
+spring.cloud.gcp.spanner.instance-id=integration-instance
```

Comment:
Can we stick to the same configuration as described in the actual sample README:
```
spring.cloud.gcp.spanner.instance-id=spring-demo
spring.cloud.gcp.spanner.database=trades
```